### PR TITLE
[hotfix]fix chatglm rmsnorm 

### PR DESCRIPTION
**Pull request by CjhHa1**
## 📌 Checklist before creating the PR

- [ ] I have created an issue for this PR for traceability
- [x] The title follows the standard format: `[doc/gemini/tensor/...]: A concise description`
- [x] I have added relevant tags if possible for us to better distinguish different PRs

## 🚨 Issue number

> Link this PR to your issue with words like fixed to automatically close the linked issue upon merge
>
> e.g. `fixed #1234`, `closed #1234`, `resolved #1234`

## 📝 What does this PR do?

> Summarize your work here.
> if you have any plots/diagrams/screenshots/tables, please attach them here.

This PR fixes the import err of chatglm rmsnorm

## 💥 Checklist before requesting a review

- [ ] I have linked my PR to an issue ([instruction](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue))
- [ ] My issue clearly describes the problem/feature/proposal, with diagrams/charts/table/code if possible
- [ ] I have performed a self-review of my code
- [ ] I have added thorough tests.
- [ ] I have added docstrings for all the functions/methods I implemented

## ⭐️ Do you enjoy contributing to Colossal-AI?

- [x] 🌝 Yes, I do.
- [ ] 🌚 No, I don't.

Tell us more if you don't enjoy contributing to Colossal-AI.

### CODE REVIEW

File: colossalai/inference/engine/policies/chatglm2.py [hunk 1]
```diff
@@ -1,29 +1,48 @@
...
-class ChatGLM2InferPolicy(ChatGLMModelPolicy):
+def get_triton_rmsnorm_forward():
+    if HAS_TRITON_RMSNORM:
+
+        def _triton_rmsnorm_forward(self: RMSNorm, hidden_states: torch.Tensor):
+            return rmsnorm_forward(hidden_states, self.weight.data, self.variance_epsilon)
+
+        return _triton_rmsnorm_forward
+    else:
+        return None
```
Comment (ver217) [inline colossalai/inference/engine/policies/chatglm2.py:42]: This should raise an exception

Comment (CjhHa1) [reply colossalai/inference/engine/policies/chatglm2.py]: fixed

**Comment by CjhHa1** [comment]
before replace rms_norm
image

after rms_norm
image

File: colossalai/inference/engine/policies/chatglm2.py [hunk 2]
```diff
@@ -56,6 +75,16 @@ def module_policy(self):
             )
         # for rmsnorm and others, we need to check the shape
 
+        infer_forward = None
+
+        if HAS_TRITON_RMSNORM:
+            infer_forward = get_triton_rmsnorm_forward()
+            if infer_forward is not None:
```
Comment (ver217) [inline colossalai/inference/engine/policies/chatglm2.py:82]: If `HAS_TRITON_RMSNORM`, the `infer_forward` won't be None

**Review by ver217 — CHANGES_REQUESTED** [review]

**Pull request merged**